docs/about/team/internships.md

```diff
@@ -4,6 +4,11 @@ sidebar_position: 4
 
 # Internships at the AI Lab
 
+### [Werbrouck, Francois](https://www.linkedin.com/in/francois-werbrouck) - Base de données Hiver 2024
+> Ayant déjà travaillé pour le gouvernement, je suis étonnament surpris de mon intégration chaleureuse avec l'équipe. Celle-ci était très guidé documenté et j'ai eu comme tâche d'améliorer le processus d'intégration pour mes futurs collègues. Bref, j'ai hâte de commencer à pouvoir participer aux projets de l'équipe puisque je me sens déjà comme l'un des leurs.
```

Comment:
this is paragraph is a single line longer than >80

Reply:
Text should now be wrapped

Reply:
Nope...

![image](https://github.com/ai-cfia/ai-cfia-ia-acia.github.io/assets/538542/a1c76888-cdac-4dd6-ba0c-b933570938e0)

Reply:
@FrancoisWerbrouck-CFIA when a fix is made, please mention it in the conversation so that the reviewer knows to validate the change.